wellcomeml/ml/keras_utils.py

```diff
@@ -1,12 +1,34 @@
 from collections import defaultdict
 
-from sklearn.metrics import f1_score, precision_score, recall_score
+import pandas as pd
 import tensorflow as tf
+from sklearn.metrics import f1_score, precision_score, recall_score
 
 
 class Metrics(tf.keras.callbacks.Callback):
```

Comment:
This is nice. I am impressed that it worked with all the other classes that used it seamlessly

Reply:
haha yes... :crossed_fingers: